src/oic/utils/authn/ldapc.py

```diff
@@ -1,4 +1,7 @@
-import ldap
+try:
+    import ldap
+except ImportError:
+    raise ImportError('This module can be used only with ldap isntalled.')
```

Comment:
isntalled -> installed. Also, should we here instead mention the package - pyldap? Might be faster for the user.